tests/test_convergence.py

```diff
@@ -0,0 +1,101 @@
+import numpy as np
+from ase import Atoms
+import pytest
+from numpy.testing import assert_allclose
+from anisoap.representations import EllipsoidalDensityProjection
+import metatensor
+
+from scipy.special import sph_harm
+
+
+class TestGaussianConvergence:
+    sigmas = np.array([1.0, 2.0, 3.0])
+
+    @pytest.mark.parametrize("sigma", sigmas)
+    def test_single_atom_isotropic_convergence(self, sigma):
+        """
+        Test that coefficients are correct, such that the approximation using these coefficients in the expansion
+        can reasonably recreate the original atomic gaussian (in this case, an unnormalized gaussian with sigma=1)
+        """
+
+        frame = Atoms(positions=np.array([[0, 0, 0]]), numbers=[0])
+        frame.arrays["quaternions"] = np.array([[1, 0, 0, 0]])
+
+        frame.arrays[r"c_diameter[1]"] = 2 * sigma * np.ones(1)
+        frame.arrays[r"c_diameter[2]"] = 2 * sigma * np.ones(1)
```

Comment:
where is c_diameter []? Why are you also multiplying by np.one(). Wouldn't it just be the same value? Why is this necessary?

Reply:
c_diameter[1], c_diameter[2], c_diameter[3] are the dimensions of the ellipsoid.

Multiplying by np.ones(1) looks weird, but I'm multiplying by np.ones(n) because each of the c_diameter fields needs to be length n, where n is the number of ellipsoids in the frame. Here, I'm just testing convergence on a single particle, but later, I may want to test convergence on multiple particles.